.env.example

```
@@ -12,10 +18,11 @@ REDIS_PASSWORD=
 JWT_SECRET=
 JWT_ACCESS_TOKEN_EXPIRATION_MINUTES=30
 JWT_REFRESH_TOKEN_EXPIRATION_DAYS=14
+# HTTPS 운영 배포에서는 true를 사용하고, 로컬 또는 임시 HTTP 개발 환경에서만 false로 변경합니다.
 JWT_REFRESH_TOKEN_COOKIE_SECURE=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_REFRESH_TOKEN_COOKIE_SECURE key should go before the JWT_REFRESH_TOKEN_EXPIRATION_DAYS key

(UnorderedKey)